subjects/gaming/2.5d-adventure/audit/README.md

```diff
@@ -0,0 +1,89 @@
+> Due to file size reason, the solution might be uploaded on GitHub instead of Gitea!
+
+#### General
+
+##### If, at any point during the audit, coupled code is detected that could have been resolved using the Observer pattern (`event dispatchers`), the audit will stop, and the student will fail the project.
+
+###### Is the game following a coherent theme ?
+
+#### Main Menu
+
+###### Is the main menu widget on a separate level/map from the main game map?
+
+###### Does the main menu contain a Start game button?
+
+###### Does the main menu contain an Exit game button?
+
+#### Player Character
+
+###### Does the player character have a skeletal mesh?
+
+###### Can the player character move only along two axes: left-right and up-down?
+
+###### Does the player character have a basic locomotion system with all the required animations?
+
+###### Does the player character transition smoothly between walking and running animations based on their speed?
+
+#### Collectible
+
+###### Does the collectible Actor have a static mesh?
+
+###### Does the collectible rotate around an axis?
+
+###### Does the collectible have a box collider that acts as a trigger?
+
+###### Is the collectible collected when the player enters its box trigger?
+
+###### Does the collectible play a sound when collected?
+
+#### HUD
+
+###### Does the HUD widget display a value related to the `collectible`?
+
+###### Is the HUD updated each time a `collectible` is picked up?
+
+###### Is the HUD also updated when picking a `collectible` that spawned from killing an enemy?
```

Comment:
This part is not mentioned in the subject, we should add it otherwise the student dont know about it

Reply:
okay, i thought it was clear that picking up a collectible -whatever its source-  should update the HUD.
but sure i will explicitly ask them to do that.

Reply:
that is clear, the part of the `collectible` being spawned from killing is the part not mentioned there, it is mentioned in other section, you can also move the question down to that section, it is up to you, let me know what you think its better

Reply:
i mentioned it in the enemy section as its related to it.
i think i will add the following under where im asking them to spawn it in the subject:

> Tip: Make sure the instance of `collectible` spawned from killing an enemy also updates the HUD.